plugins/goodix-fp/goodixfp.quirk

```diff
@@ -0,0 +1,36 @@
+# Goodix Fingerprint sensor
+[DeviceInstanceId=USB\VID_27C6&PID_60A2]
+Plugin = goodixfp
+Summary = An open source goodix Fingerprint sensor
+FirmwareSizeMin = 0x20000
+InstallDuration = 10
+
+[DeviceInstanceId=USB\VID_27C6&PID_6384]
+Plugin = goodixfp
+Summary = An open source goodix Fingerprint sensor
+FirmwareSizeMin = 0x20000
+InstallDuration = 10
+
+[DeviceInstanceId=USB\VID_27C6&PID_639C]
+Plugin = goodixfp
+Summary = An open source goodix Fingerprint sensor
+FirmwareSizeMin = 0x20000
+InstallDuration = 10
+
+[DeviceInstanceId=USB\VID_27C6&PID_63AC]
+Plugin = goodixfp
+Summary = An open source goodix Fingerprint sensor
+FirmwareSizeMin = 0x20000
+InstallDuration = 10
+
+[DeviceInstanceId=USB\VID_27C6&PID_6594]
+Plugin = goodixfp
+Summary = An open source goodix Fingerprint sensor
+FirmwareSizeMin = 0x20000
+InstallDuration = 10
+
+[DeviceInstanceId=USB\VID_27C6&PID_6496]
+Plugin = goodixfp
+Summary = An open source goodix Fingerprint sensor
```

Comment:
I think the summary should move to fu_goodixfp_device_init if it's all the same. Also I don't think the Fingerprint sensor is open source, right? You probably want something like `Fingerprint sensor` as the Vendor is already set to `Goodix`